contracts/.env.example

```diff
@@ -0,0 +1,5 @@
+ETH_RPC_URL="https://eth-mainnet.g.alchemy.com/v2/<YOUR_API_KEY>"
+
+V4_POOL_MANAGER=""
+ANGSTROM_MULTISIG=""
+ANGSTROM_ADDRESS_TOKEN_ID=""
```

Comment:
what is this?

Reply:
the token_id

Reply:
So sub zero basically has you first mint a token corresponding to the vanity address. It's basically the salt.